com.unity.perception/Documentation~/SetupSteps.md

```diff
@@ -6,23 +6,18 @@
 Below are two options for getting started using the Perception package. Option 1 is opening existing test projects in the repository. Option 2 new Unity project and integrate the Perception package.
 
 ### Option 1: PerceptionHDRP & PerceptionURP Projects
-The repository includes two projects for local development in `TestProjects` folder, one set up for HDRP and the other for URP. You can open these with the Unity
-editor you installed in Setup instructions.
+The repository includes two projects for local development in `TestProjects` folder, one set up for HDRP and the other for URP. These can be opened in by adding them to the Unity Hub.
 
 <img src="images/TestProjects.PNG" align="middle"/>
 
 ### Option 2: Create a new Project 
-These option is walkthrough in creating a new project, then adding the Perception SDK package to the project for development use.
+You can also set up a new or existing project to use Perception.
 *The following instructions reference the Unity doc's page on [installing a local package](https://docs.unity3d.com/Manual/upm-ui-local.html)*
 
-#### Create a new project 
-1. Create a new HDRP project or open an existing project
-	1. Creating anew HDRP project can be done by creating a new project using the HDRP template 
-2. Back in Unity editor, got Window ->  Package Manager
-	1. Add the High Definition RP package, version 7.1.2 or later from the packages list 
-	2. In the Package Manager window find and click the ***+*** button in the upper lefthand corner of the window
-	3. Select the ***add package from disk*** option
-	4. Navigate to the com.unity.perception folder in your cloned repository and select the package.json file
-3. Once you have a project with Perception SDK installed you can move forward to the Getting Started walkthrough 
+1. Create a new HDRP or URP project, or open an existing project
+2. Open `Window` ->  `Package Manager`
+	1. In the Package Manager window find and click the ***+*** button in the upper lefthand corner of the window
+	2. Select the ***add package from disk*** option
+	3. Navigate to the com.unity.perception folder in your cloned repository and select the package.json file
 
-Once completed you can move on to the getting started steps, click [here](Documentation~/GettingStarted.md) to start project setup.
\ No newline at end of file
+Once completed you can continue with the [getting started steps](GettingStarted.md).
```

Comment:
(GettingStarted.md). This appears to be a broken link, I you need to have Documentation~/GettingStarted so git knows the file path to the md

Reply:
It works on GitHub. Which markdown viewer are you using? https://github.com/Unity-Technologies/com.unity.perception/blob/update-setup-steps/com.unity.perception/Documentation~/SetupSteps.md